README.rst

```diff
@@ -129,7 +129,7 @@ page or ask us at our active Gitter channel, where we have maintainers from
 all over the world. We appreciate any help!
 
 We also have a
-`newcomer guide <http://coala.io/newcomer>`__
+`Guide for Newcomers <http://coala.io/newcomer>`__
```

Comment:
This change does not make sense to me 😅 Sorry I have to close this PR.

If you think there is a typo or potential improvement, please open an issue first instead of creating a pull request directly.

Reply:
I am trying to address the Issue https://github.com/coala/coala-bears/issues/2838
Please correct me if I have misunderstood what the issue says.

Reply:
Sorry, I was not aware of that. Please address what issue you are solving both in your commit message and in the opening comment of this PR.

Reply:
I have done the suggested changes.